example/chrome-example.php

```diff
@@ -4,12 +4,37 @@
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-// @TODO Change me
-$url = 'ws://127.0.0.1:9222/devtools/browser/b1c46ece-85c4-48b5-850d-687296d0566e';
 
-\Amp\Loop::run(function () use($url) {
+\Amp\Loop::run(function () {
     try {
-        $logger = new \Symfony\Component\Console\Logger\ConsoleLogger(new \Symfony\Component\Console\Output\ConsoleOutput(\Symfony\Component\Console\Output\OutputInterface::VERBOSITY_NORMAL));
+        $process = new \Amp\Process\Process(
+            [
+                "google-chrome-stable",
```

Comment:
I think this should be `google-chrome` instead, which should also exist if `google-chrome-stable` exists.

Reply:
Depends on the env, will refactor this latter, right now i just want something that work on my differents env, so i can try / work on the api